src/main/java/tutorease/address/logic/commands/FindContactCommand.java

```diff
@@ -8,24 +8,26 @@
 import tutorease.address.model.person.NameContainsKeywordsPredicate;
 
 /**
- * Finds and lists all persons in address book whose name contains any of the argument keywords.
- * Keyword matching is case insensitive.
+ * Finds all persons whose name is contained in the keyword.
  */
-public class FindCommand extends Command {
+public class FindContactCommand extends Command {
+    public static final String COMMAND_WORD = "contact";
 
-    public static final String COMMAND_WORD = "find";
+    public static final String SUB_COMMAND_WORD = "find";
```

Comment:
won't need a SUB_COMMAND_WORD if follow the suggestion above. will just be ContactCommand.COMMAND_WORD and COMMAND_WORD